rfc6962-bis.xml

```diff
@@ -1415,10 +1430,7 @@ but it is expected there will be a variety.
                 Remove the Transparency Information extension described in <xref target="x509v3_transinfo_extension"/>
               </t>
               <t>
-                Remove the Redacted Labels extension described in <xref target="redaction_extension"/>
-              </t>
-              <t>
-                Replace leftmost labels of each DNS-ID with <spanx style="verb">?</spanx>, based on the INTEGER value corresponding to that DNS-ID in the extension.
+                If the <xref target="redacted_san_extension">Redacted SAN</xref> extension is present, remove the subjectAltName extension.
```

Comment:
I agree there should be a description.  I also think that at least some amount of verification should be required of clients.  We don't want to allow a pre-certificate for `?.example.com` to be used with `www.example.org` for instance.

Here's some suggested text:

> If the Redacted SAN extension is present, TLS clients MUST check that the SAN extension is present, that the SAN extension contains the same number of identifiers as the Redacted SAN extension, and that each identifier in the SAN extension has a matching identifier at the same position in the Redacted SAN extension.  Two identifiers are matching if either:
>
> * The two identifiers are identical; or,
>
> * Both identifiers are DNS names, have the same number of labels, and each label in the SAN identifier has a matching label at the same position in the Redacted SAN identifier.  Two DNS labels are matching if either:
>
>   * The two labels are identical; or,
>
>   * The label from the the Redacted SAN identifier is equal to REDACT(label from SAN identifier) [cross-reference to redacting_labels]
>
> If any of these checks fail, the certificate MUST NOT be considered compliant.